relay-system/src/controller.rs

```diff
@@ -55,11 +58,22 @@ pub use actix::actors::signal::{Signal, SignalType};
 ///     Ok(())
 /// }).unwrap();
 /// ```
+
+type ShutdownReceiver = watch::Receiver<Option<Shutdown>>; // Find a good name for this
+type ShutdownSender = watch::Sender<Option<Shutdown>>; // Check of that is really needed
+                                                       // Only used once but gives a nice symetry
```

Comment:
I'm kind of in favour of not using type aliases for such small types.  If you'd prefer to keep them give them doc comments (and cleanup the comments), I think the names are fine.